### DISCUSSION TITLE
iOS 16 Lock Screen widget

### DISCUSSION THREAD

**Discussion by OpenSource03** [community/community#33841] [Mobile]
Please consider adding Lock Screen contributions widget, similar to the one on the Home Screen. Would be very cool!

Thanks!

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.81.0 (78683238)
Device: iPhone 12 (iOS 16.1)
</details>

**Comment by eliperkins** [comment]
Hey @OpenSource03 , thanks for this feedback! Having a contribution graph widget on the Lock Screen _does_ sound super neat. Let me see what I can do!

Can I ask some questions to do a little research here? 😎

- As far as I understand it, Lock Screen widgets will be in grayscale/monochrome. One of the most powerful ways that the contribution graph conveys information is through the colors of the squares. How might you expect to differentiate different levels of contribution on a monochrome display like this?
- What other Lock Screen widgets would you like to see? 😇

**Reply by OpenSource03** [reply]
Hi!

Thanks for your interest and response.

As for how to implement it, my idea came from a video from MKBHD, where he reviewed the latest iPhone. Check out the Lock Screen widget he had.

![image](https://user-images.githubusercontent.com/29690431/191993530-f3efeff7-fec8-42b6-8cfc-ab890ce5dea4.jpeg)

I believe something in this style would look absolutely amazing!

As for the other widgets... Well... I don't have many ideas that would fit into the small portion of the Lock Screen. Maybe some other people around here may have some?

**Reply by SunLyEak** [reply]
[](https://github.com/SunLyEak/-.git)

**Comment by auvrwhiite** [comment]
> Please consider adding Lock Screen contributions widget, similar to the one on the Home Screen. Would be very cool!
> 
> Thanks!
> 
> <details>
> <summary>ℹ️ Additional Debug Info</summary>
> GitHub for iOS, Version 1.81.0 (78683238)
> Device: iPhone 12 (iOS 16.1)
> </details>

**Comment by hfhbd** [comment]
There is finally a Lock Screen widget. Unfortunately, you can't change the filter like the Home Screen Widget. It always shows the number of created PRs. Personally, I like the number of assigned/review requested more because that's the number of my todos.

**Reply by OpenSource03** [reply]
Actually, if you click on the widget, you'll see the filter option.

**Reply by hfhbd** [reply]
Not on the Lock Screen 🤔
Oh you are right. You need two clicks 🤦🏻‍♂️
